index.html

```diff
@@ -0,0 +1,203 @@
+                            Get opportunities to network with CC affiliates around the world, also after the summit.
+                        </p>
+                      </article>
+                    </div>
+                    <button type="button" name="join-us" class="btn main-btn">Join the 2023 conference!</button>
+                    <a class="see-program"><u>SEE THE WHOLE PROGRAM</u></a>
+                    </section>
+                    <section class="featured-speakers flex-column-center" id="speakers">
+                        <h2 class="featured-speakers-header light-black heading-medium">This Years Conference Speakers</h2>
+                        <hr class="horizontal-line text-center" />
+                        <div class="speakers-list"></div>
+                        <button type="button" name="join-us" class="btn speakers-btn" onclick="toggleSpeakers()">
+                          MORE <i class="fas fa-angle-down down-arrow"></i>
+                        </button>
+                      </section>
+
+                      <section class="partners-container-desktop">
+                    <section class="partners-desktop bg-light-black flex-column-center" id="partners-desktop">
+                        <h2 class="partners-desktop-heading light-grey heading-medium">This Years Partners</h2>
+                        <hr class="horizontal-line text-center" />
+                        <ul class="partners-desktop-list">
+                          <li class="list-item-desktop-container">
+                            <img class="list-item-desktop" src="./images/partners/laxact-logo.png" alt="Logo of laxACT" />
+                            </li>
+
+                            <li class="list-item-desktop-container">
+                                <img class="list-item-desktop" src="./images/partners/formuler.png" alt="Logo of Formuler" />
+                                </li>
+
+                              <li class="list-item-desktop-container">
+                                <img class="list-item-desktop" src="./images/partners/jadoo.png" alt="Logo of Jadoo TV"/>
+                                </li>
+
+                              <li class="list-item-desktop-container">
+                                <img class="list-item-desktop" src="./images/partners/buzztv.png" alt="Logo of Buzz TV" />
+                                </li>
+
+                              <li class="list-item-desktop-container">
+                                <img class="list-item-desktop" src="./images/partners/apollo.png" alt="Logo of Apollo Global" />
+                                </li>
+
+                              <li class="list-item-desktop-container">
+                                <img class="list-item-desktop" src="./images/partners/iptvsmarters.png" alt="Logo of IPTV Smarters" />
+                            </li>
+                        </ul>
+                        </section>
```

Comment:
- [ ] @katarighe I can't see your patient section on the desktop view, kindly add it and ensure you implement the change previously requested for this section

![image](https://github.com/katarighe/microverse-capstone-project-1/assets/61408860/3d93289e-e4d7-447f-9467-0a340db58593)